Automattic/woocommerce-payments
# Add warning to advanced settings

#### Changes proposed in this Pull Request
This PR adds a warning about incompatible extensions in the WooPay advanced settings page and fixes the icons being used all over the admin area

#### Testing instructions
- Checkout payments server to the latest version
- Change [this line](https://github.com/Automattic/woocommerce-payments-server/blob/trunk/server/wp-content/rest-api-plugins/endpoints/wcpay/woopay/class-woopay-controller.php#L60) to look like this:
`'incompatible_extensions' => ['test-extension', 'test-extension-2'],` 
- On your merchant site
- Execute the `validate_incompatible_extensions` cron
- Install and activate this extension
[test-extension.zip](https://github.com/Automattic/woocommerce-payments/files/11791369/test-extension.zip)
- Go to /wp-admin > Payments > Scroll to Express checkout
- Make sure there is a warning about incompatible extensions showing up
- Click the WooPay customize button
- Make sure there is a warning about incompatible extensions showing up
- Disable the extension and make sure the warning is gone in both places



The warnings should look like this:
**Settings page**
<img width="1088" alt="Screenshot 2023-07-05 at 5 56 28 PM" src="https://github.com/Automattic/woocommerce-payments/assets/6342964/6f0ffe6d-ef7c-4c62-9ac3-a8093dcbcfe5">
<img width="366" alt="Screenshot 2023-07-05 at 5 56 44 PM" src="https://github.com/Automattic/woocommerce-payments/assets/6342964/1e33be2c-13ae-422e-ae13-3bf9b1e5a928">

**Advanced Settings**
<img width="1069" alt="Screenshot 2023-07-05 at 5 56 01 PM" src="https://github.com/Automattic/woocommerce-payments/assets/6342964/34a1dfa8-66b6-4201-b7df-64a022b4bb08">
<img width="329" alt="Screenshot 2023-07-05 at 5 55 55 PM" src="https://github.com/Automattic/woocommerce-payments/assets/6342964/4e55d253-dab8-4311-a5dd-a384c9ba266f">


- Go to settings page
- Enable "Issue an authorization on checkout, and capture later"
- Make sure the payment methods look like this

<img width="654" alt="Screenshot 2023-07-05 at 6 35 50 PM" src="https://github.com/Automattic/woocommerce-payments/assets/6342964/ac6c30db-782e-45b8-85b5-a3b88fac3de9">




-------------------


**Post merge**
